src/tutorials/region-migration.md

```diff
@@ -0,0 +1,55 @@
+# Region migration
+
+Platform.sh is available in a number of different *Regions*.  Each region is a self-contained copy of Platform.sh in a single datacenter.  When you first create a project you can specify what region it should be in.
```

Comment:
em... i'm not sure .... `what region` v.s. `which region`?

Reply:
I think which is probably more correct, although both are colloquially common.  I'll fix.